version

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Version script for Marathon.
+#
+# Change MINOR to x+1 and BRANCH_POINT to commit hash of common ancestor of master and releases/1.x
+# after a releases/1.x was cut
+MAJOR=1
+MINOR=6
+BRANCH_POINT=aa89e89
+
+# Infer version
+# Number of commits since branch point
+BUILD_NUMBER="$(git rev-list --count --first-parent $BRANCH_POINT..HEAD)"
```

Comment:
Why not `git rev-list --count HEAD ^master`?

Reply:
Preference to name this variable "COMMIT_NUMBER"

I like the script. The divergence point isn't always discernible by the master branch, right?

Reply:
> Why not `git rev-list --count HEAD ^master`?

We don't always run this on master. And Tim is correct, it is not always trivial to find divergence points from branches.